src/App.js

```diff
@@ -28,5 +28,37 @@ Feel free to have fun and experiment with others too!
 If you get stuck or need help, please reach out on Slack.
 */
 export default function App() {
-  return <div>Replace this with your beautiful JSX</div>;
+  return (
+    <div className="App">
+      <header className="App-header">
+        <img src={logo} className="App-logo" alt="logo" />
+        <h1>Redi React Fall 2020 Team</h1>
+      </header>
+      <main>
+      <table className="members-table">
+          <thead>
+            <tr>
+              <th>Photo</th>
+              <th>Name</th>
+              <th>Role</th>
+            </tr>
+          </thead>
+          <tbody>
+            {team.map((team) => (
+              <tr className="member-row" key={team.name}>
+                <td className="avatar-cell">
+                  <img alt="memberPhoto"src={team.image} />
+                </td>
+                <td className="name-cell">{team.name}</td>
+                <td className="role-cell">{team.role}</td>
```

Comment:
For readability, it could be nice to use `member` or `teamMember` for the individual variable name. That way it shows that each of these iterations represents one team member.

i.e.
```javascript
team.map((member) => (

team.map((teamMember) => (
```